etna/datasets/tsdataset.py

```diff
@@ -288,6 +288,7 @@ def make_future(self, future_steps: int) -> "TSDataset":
 
         if self.transforms is not None:
             for transform in self.transforms:
+                tslogger.log(f"Transform {transform.__class__.__name__} is applied to dataset")
```

Comment:
why not `transform.__repr__()`?
if you have smth like `transforms=[LagTransform(in_column="target", ...), LagTransform(in_column="exog", ...)]` there are two different transforms of the same class

Reply:
Here I just repeat code from `TSDataset.transform`. If wee want this, I can change it in both places and in `TSDataset.fit_transform` too.